### DISCUSSION TITLE
SIEMENS Inhab Smart Home Energy Monitor compatibility?

### DISCUSSION THREAD

**Discussion by lima-faux** [emporia-vue-local/esphome#324] [Q&A]
Would the SIEMENS Inhab Smart Home Energy Monitor be compatible with this project ?

https://www.siemens.com/us/en/products/energy/low-voltage/new-homes/inhab/inhab-energy-monitor.html

**Comment by flaviut** [comment]
It is not compatible.

**Comment by CavemanTitan** [comment]
> Would the SIEMENS Inhab Smart Home Energy Monitor be compatible with this project ?
> 
> 
> 
> https://www.siemens.com/us/en/products/energy/low-voltage/new-homes/inhab/inhab-energy-monitor.html
> 
> 

It does look to be an exact copy of the Emporia Vue Gen 3. Their app also looked similar.

I'm curious to see if it will be supported easily.

**Reply by flaviut** [reply]
You are right… I should have looked more closely but it does look like a white label Vue 3.

I think buying one and seeing what's inside is fairly low risk, and you can make a backup before flashing so you can restore it if things don't work out.
